source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -158,15 +158,15 @@ class Driver : public Tracing::Driver {
  * Information about the Zipkin collector.
  */
 struct CollectorInfo {
-  // The Zipkin collector endpoint/path to receive the collected trace data. e.g. /api/v1/spans if
-  // HTTP_JSON_V1 or /api/v2/spans otherwise.
+  // The Zipkin collector endpoint/path to receive the collected trace data. e.g. default is
+  // /api/v2/spans.
   std::string endpoint_{DEFAULT_COLLECTOR_ENDPOINT};
 
   // The version of the collector. This is related to endpoint's supported payload specification and
-  // transport. Currently it defaults to envoy::config::trace::v3::ZipkinConfig::HTTP_JSON_V1. In
+  // transport. Currently it defaults to envoy::config::trace::v3::ZipkinConfig::HTTP_JSON. In
   // the future, we will throw when collector_endpoint_version is not specified.
   envoy::config::trace::v3::ZipkinConfig::CollectorEndpointVersion version_{
-      envoy::config::trace::v3::ZipkinConfig::hidden_envoy_deprecated_HTTP_JSON_V1};
+      envoy::config::trace::v3::ZipkinConfig::HTTP_JSON};
```

Comment:
If there is no PGV annotation in `zipkin.proto`, this means that the default is the enum 0 value, which is `hidden_envoy_deprecated_HTTP_JSON_V1` and it will overwrite this anyway.

Reply:
Yep, as we discussed offline. We want to fire the exception if the `CollectorEndpointVersion` is not specified in the config